ante/ante.go

```diff
@@ -130,9 +136,12 @@ func newCosmosAnteHandler(opts HandlerOptions) sdk.AnteHandler {
 	}
 
 	anteDecorators := []sdk.AnteDecorator{
-		evmante.RejectMessagesDecorator{}, // reject MsgEthereumTxs
+		cosmosante.NewRejectMessagesDecorator(), // reject MsgEthereumTxs
 		// disable the Msg types that cannot be included on an authz.MsgExec msgs field
-		evmante.NewAuthzLimiterDecorator(disabledAuthzMsgs),
+		cosmosante.NewAuthzLimiterDecorator(
+			sdk.MsgTypeURL(&evmtypes.MsgEthereumTx{}),
+			sdk.MsgTypeURL(&sdkvesting.MsgCreateVestingAccount{}),
```

Comment:
The vesting module was removed at this time.

https://github.com/xpladev/xpla/pull/136/files